src/keyfile.c

```diff
@@ -1287,13 +1285,6 @@ void configuration_apply_settings(void)
 	}
```

Comment:
This part works also before main window realisation but there's no problem having it here.